src/main/java/lotto/domain/WinningNumbers.java

```diff
@@ -0,0 +1,26 @@
+package lotto.domain;
+
+import static lotto.constant.ErrorMessage.WINNING_NUMBERS_CONTAIN_BONUS_NUMBER;
+
+public class WinningNumbers {
+    private final Lotto winningLotto;
+    private final LottoNumber bonusNumber;
+
+    public WinningNumbers(Lotto lotto, int bonusNumber) {
+        LottoNumber generatedBonusNumber = new LottoNumber(bonusNumber);
+        validateBonusNumber(lotto, generatedBonusNumber);
+        this.winningLotto = lotto;
+        this.bonusNumber = generatedBonusNumber;
+    }
```

Comment:
생성자 오버로딩을 활용해서 아래와 같이 변경해보면 어떨까요? 😃 

```suggestion
    public WinningNumbers(Lotto lotto, LottoNumber bonusNumber) {
        validateBonusNumber(lotto, bonusNumber);
        this.winningLotto = lotto;
        this.bonusNumber = bonusNumber;
    }

    public WinningNumbers(Lotto lotto, int bonusNumber) {
        this(lotto, new LottoNumber(bonusNumber));   
    }
```